Install instructions update and clarification (#43)
osrf/ros2multirobotbook: src/intro.md

The open-source and freely distributable [simulation assets](https://app.ignitionrobotics.org/fuel) are created and shared to accelerate simulation efforts.

## Installation of the RMF Essentials

> The current version of RMF is 1.0.2 and it targets ROS Eloquent with Debian binary packages released for Ubuntu Bionic 18.04 LTS.

Reviewer: ```suggestion
The current version of RMF is 1.0.2. It targets ROS Eloquent, and Debian binary packages are released for Ubuntu Bionic 18.04 LTS.
```

> RMF leverages the ROS and Gazebo-Ignition ecosystem so we will have to setup their repositories as the first steps of the RMF installation process. Since the current binary release targets Ubuntu Bionic please ensure you match the system and version before proceeding with the installation. If you have ROS 2 and Gazebo installed you can directly skip to the [Setup Sources and Installation of RMF](#setup-sources-and-installation-of-rmf) section.

Reviewer: ```suggestion
RMF leverages the ROS and Gazebo-Ignition ecosystem. We will have to setup their repositories as the first steps of the RMF installation process. Since the current binary release targets Ubuntu Bionic please ensure you match the system and version before proceeding with the installation. If you have ROS 2 and Gazebo installed you can directly skip to the [Setup Sources and Installation of RMF](#setup-sources-and-installation-of-rmf) section.
```

sudo apt-get update
```

Finally, the following installs a basic set of packages that should get you going with RMF:

```
sudo apt-get install ros-eloquent-ament-cmake-catch2 ros-eloquent-building-gazebo-plugins ros-eloquent-building-map-msgs ros-eloquent-building-map-tools ros-eloquent-rmf-cmake-uncrustify ros-eloquent-rmf-dispenser-msgs ros-eloquent-rmf-door-msgs ros-eloquent-rmf-fleet-adapter ros-eloquent-rmf-fleet-msgs ros-eloquent-rmf-lift-msgs ros-eloquent-rmf-task-msgs ros-eloquent-rmf-traffic-msgs ros-eloquent-rmf-traffic-ros2 ros-eloquent-rmf-traffic ros-eloquent-rmf-utils ros-eloquent-traffic-editor
```

## Install and run RMF demos

You can install the provided RMF demos from their Debian package:

```
sudo apt-get install ros-eloquent-demos
```

> If ROS 2 is not installed using at least `ros-eloquent-ros-base` you will also need the following packages:

Reviewer: I think this would work better at the end of the "Setup ROS Sources" section. Because you have told readers who already have ROS installed to skip that section, it makes sense to assume those who need to set up their sources also need to install the necessary ROS packages.

Author: Will move it, thanks.

ros2 launch demos airport_terminal.launch.xml
```

> The airport world was created to not spawn the robots when launched. Instead there are "RobotPlaceholder" models where they get dynamically spawned when you run:

Reviewer: ```suggestion
The airport world was created to not spawn the robots when launched. Instead there are "RobotPlaceholder" models where they get dynamically spawned. To spawn the robots, run this command:
```
